src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx

```diff
@@ -140,6 +139,17 @@ To configure the integration, edit the config in the integration's YAML config f
 
 To read all about these common settings, refer to our [configuration format](/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format/#configuration-basics) document.
 
+### Permissions [#permissions]
+The `username` set in the integration config needs to be granted some permissions in order to monitor a IBM MQ infrastructure.
+The required permissions varies depending on monitored queues, channels, and topics.
+
+In general, the user should be able to:
+ - connect to the queue manager having `DISPLAY` and `INQUIRY` enabled.
+ - access the channel specified in the integration config (by default `SYSTEM.DEF.SVRCONN`)
+ - operate priviledge commands on queues `SYSTEM.DEFAULT.MODEL.QUEUE` and `SYSTEM.ADMIN.COMMAND.QUEUE` having the `Display` permission.
+ - access with `Display` and `Subscribe` for the topics `SYSTEM.ADMIN.TOPIC` and `SYSTEM.BROKER.ADMIN.STREAM`
```

Comment:
these are the permissions required, however some additional one are needed depending on their specific setup